base/templates/base/achats.html

```diff
@@ -63,7 +63,7 @@ <h2>{{ household }} - <small>Solde : {{ household.account }} € </small></h2>
         </tr>
       </tbody>
     </table>
-    <div class="text-right">
+    <div class="text-right" id="list-overlay-actions">
```

Comment:
```suggestion
    <div class="text-right list-overlay-actions">
```
Ça marche mieux comme ça non ?  :-)